HousingFinanceInterimApi/V1/Infrastructure/DatabaseContext.cs

```diff
@@ -47,6 +47,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<UPCashDump>().Property(x => x.Timestamp).HasDefaultValueSql("GETDATE()");
             modelBuilder.Entity<UPHousingCashDump>().Property(x => x.Timestamp).HasDefaultValueSql("GETDATE()");
             modelBuilder.Entity<AdjustmentAux>().Property(x => x.Timestamp).HasDefaultValueSql("GETDATE()");
+            modelBuilder.Entity<MAProperty>().HasKey(x => x.PropRef);
+            modelBuilder.Entity<UHProperty>().HasKey(x => x.PropRef);
```

Comment:
I think it's a keyless table within the database _(no PK constraint)_, but let's give it a go.

Reply:
Fome some reason I got Entity Framework errors when I specified NoKey in modelBuilder

Reply:
In the Python we needed to specify a compound key to avoid the ORM selecting/deleting the wrong records for these

Reply:
@adamtry which columns were used to specify the compound key?

Reply:
Ah actually for these property tables we just used the propref, my bad. Yeah, these should in theory have one unique row per propref.

For the UHMini and SSMini tables though we needed a combo of propref, tenancy agreement ref, period number, and year